plotter/labels.go

```diff
@@ -96,11 +96,16 @@ func (l *Labels) DataRange() (xmin, xmax, ymin, ymax float64) {
 // plot.GlyphBoxer interface.
 func (l *Labels) GlyphBoxes(p *plot.Plot) []plot.GlyphBox {
 	bs := make([]plot.GlyphBox, len(l.Labels))
+	offset := vg.Point{X: l.XOffset, Y: l.YOffset}
```

Comment:
This sort of suggests that `Labels.[XY]Offset` should have been a `vg.Point`. Maybe later?

Reply:
I'll see if I can swizzle that in front of this commit.